.github/workflows/ci.yml

```diff
@@ -103,6 +103,9 @@ jobs:
       run:
         shell: bash
 
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+
```

Comment:
I would rather not set `GITHUB_TOKEN` for the whole job. Please instead just set it for the "Test" step as I suggested in https://github.com/trailofbits/dylint/pull/1622#issuecomment-2854592478.